docs/aerodromes/Alice.md

```diff
@@ -12,12 +12,13 @@
 | Alice Springs ATIS | N/A | 123.000 | YBAS_ATIS |
 
 ## Airspace
-
 <figure markdown>
 ![YBAS Airspace](img/ybas_airspace.jpg){ width="700" }
   <figcaption>YBAS Airspace</figcaption>
 </figure>
 
+AS ADC is responsible for the Class D airspace SFC-A045, as well the Class C airspace A045-A065, within the AS CTR.
```

Comment:
AS ADC is responsible for the Class D airspace `SFC` to `A045`, as well the Class C airspace `A045` to `A065`, within the AS CTR.

Reply:
Please Block text the heights to standardise the site

Reply:
Sentence structure issue; might be missing an "as"?